### ISSUE TITLE
Make mobile menu button more accessible

### ISSUE THREAD

**Issue by herrherrmann** [technologiestiftung/kulturdaten-website#40]
Our burger menu button (🍔) in the homepage header is currently missing some extra HTML attributes for accessibility – At least [`aria-expanded`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) and [`aria-controls`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls). The menu element should also get an `id` that is referred to via `aria-controls`.
